package.json

```diff
@@ -12,6 +12,7 @@
     "prelint": "tsc --noEmit",
     "postlint": "npm run skeleton:verify",
     "skeleton:apply": "code-skeleton apply",
+    "preskeleton:verify": "npm run prepack",
```

Comment:
to verify the skeleton we have to have the .js file, so before attempting the verify script we go ahead and build the typescript